segments/weather.sh

```diff
@@ -160,6 +160,10 @@ __get_condition_symbol() {
 			#echo "﹌"
 			echo "〰"
 			;;
+        "breezy")
```

Comment:
ohh, i have expand tab set in vim, this looked fine in my local copy, will update this request with tabs